examples/with-react-foundation/src/pages/index.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link'
+import { Article } from '../components/Article'
+import { Header } from '../components/Header'
+import { Callout } from 'react-foundation'
+
+// Add Foundation to index.js
+import 'foundation-sites/dist/css/foundation.min.css'
```

Comment:
Why not in `_app`? I assume other pages that want to use `react-foundation` would need this as well.

Reply:
Thanks, I have done this